### PR TITLE
mgr/telemetry: add pool_name in per pool

### PR DESCRIPTION
**Pull request by zy751713126**
Signed-off-by: Zheng Yin <zhengyin@cmss.chinamobile.com>

### CODE REVIEW

**Comment by zy751713126** [comment]
retest this please.

**Comment by zy751713126** [comment]
jenkins test make check

File: src/pybind/mgr/telemetry/module.py [hunk 1]
```diff
@@ -384,6 +384,7 @@ def compile_report(self, channels=[]):
                 report['pools'].append(
                     {
                         'pool': pool['pool'],
+                        'pool_name': pool['pool_name'],
```
Comment (tchaikov) [inline src/pybind/mgr/telemetry/module.py]: i am not sure if we should include the information which could be used to identify our users in the telemetry report.

**Comment by liewegas** [comment]
We deliberately left the pool name off as it doesn't communicate anything specifically useful to us (e.g., about what the pool is being used for) and may include identifying information.

**Pull request closed without merging**